ipaserver/install/ipa_backup.py

```diff
@@ -455,10 +460,21 @@ def db2ldif(self, instance, backend, online=True):
                     '-a', ldiffile]
             result = run(args, raiseonerr=False)
             if result.returncode != 0:
-                logger.critical('db2ldif failed: %s', result.error_log)
+                raise admintool.ScriptError(
+                    'db2ldif failed: %s '
+                    'Check if destination directory %s has enough space.'
+                    % (result.error_log, os.path.dirname(ldiffile))
+                )
 
         # Move the LDIF backup to our location
-        shutil.move(ldiffile, os.path.join(self.dir, ldifname))
+        try:
+            shutil.move(ldiffile, os.path.join(self.dir, ldifname))
+        except Exception as e:
```

Comment:
Just comment, do we really want to have regular Exception here? Would not be better not to catch all of them?
Maaaybe OSError or IOError would be enough, or even shutil.Error.

Reply:
shutil would fail with FileNotFoundError in copyfile. FileNotFoundError is a subclass of OSError.

Reply:
Are generic errors handled and visible on the command line? If not, then please add the specific and also the generic error handling.

Reply:
hi @t-woerner good idea! The last revision should address @Tiboris concern to your liking.